django/db/migrations/serializer.py

```diff
@@ -258,6 +258,11 @@ def serialize(self):
         return "pathlib.%s%r" % (prefix, self.value), {"import pathlib"}
 
 
+class DatabaseOnDeleteSerializer(BaseSerializer):
+    def serialize(self):
+        return f"models.{self.value.name}", {}
```

Comment:
Is this tested?

Reply:
Added tests for ```DatabaseOnDeleteSerializer``` - https://github.com/django/django/blob/a8789ab60cafc1576080bd5da200324b365659ac/tests/migrations/test_writer.py#L1005